src/nvim/api/vim.c

```diff
@@ -2451,3 +2452,33 @@ void nvim_del_user_command(String name, Error *err)
 {
   nvim_buf_del_user_command(-1, name, err);
 }
+
+/// Get the command line text.
```

Comment:
From `getcmdline()` docs: Return the current command-line.  Only works when the command line is being edited.

Is this function the same, returns the current? What does it return when command line is not active? The last one, or just errors?

From looking at the error call seems like it's the same. That should be in the function docs. :)

Reply:
> Is this function the same, returns the current? What does it return when command line is not active? The last one, or just errors?

Yes it returns the current, and just errors if the command line is not active.

> From looking at the error call seems like it's the same. That should be in the function docs. :)

👍 should I reference `getcmdline()` in the docs then (like `@see getcmdline()` or whatever), or just mention that in the actual docstring? Or both?

Reply:
I think just mention it and don't reference. Since this is in a way the replacement, or recommended function to use for the purpose now.